spec/requests/site_configuration_spec.rb

```diff
@@ -25,4 +25,32 @@
     expect(page).to have_css('h1', text: 'last1, first1')
     expect(page).to have_css('h2', text: 'Gesprächsdokumentationen')
   end
+
```

Comment:
It would be cool to check if
- After updating the terms, the `terms_of_use_accepted` gets reset for other users that have previously accepted the terms
- Checking whether the user's `terms_of_use_accepted` change from false to true on the database level

Reply:
I think the first check what you want is test on line 29 - i create admin and teacher (both with akcepted = true) then i login with admin - did some change in terms of use and save. Logout and login with teacher and check if there is showed page with term of use.

Second check - i tried to do new test in same file on line 57 - similar function but at the end i was reload teacher and check terms_of_use_accepted. Did you mean that?